.github/actions/prepare_release/action.yml

```diff
@@ -0,0 +1,29 @@
+name: prepare_release
+description: Prepare React Native release
+runs:
+  using: composite
+  steps:
+    - name: Yarn Install
```

Comment:
For reference, [this is the circleci job](https://github.com/facebook/react-native/blob/main/.circleci/configurations/jobs.yml#L1090-L1126)